src/bioregistry/data/bioregistry.json

```diff
@@ -130,6 +130,12 @@
       "prefix": "4DN",
       "uri_format": "https://data.4dnucleome.org/biosources/$1"
     },
+    "contact": {
+      "name": "Job Dekker",
+      "email": "Job.Dekker@umassmed.edu",
+      "orcid": "0000-0001-5631-0698",
+      "github": ""
```

Comment:
When a field like github is missing, let's omit it rather than adding an empty string